src/framework/pages/admin/LoginPage.java

```diff
@@ -0,0 +1,41 @@
+package framework.pages.admin;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import framework.selenium.SeleniumDriverManager;
+
+/**
+ * 
+ * @author Administrator
+ *
+ */
+public class LoginPage {
+
+	@FindBy(xpath = "//button") 
+	WebElement signinBtn;
+
+	private String baseUrl = "http://172.20.208.177:4046/admin/#/login";
```

Comment:
This variable should come from appConfigConstants class:
delete this line and modify line 28 driver.get(URL_ADMIN);